src/System.Runtime.Extensions/tests/System/Environment.SetEnvironmentVariable.cs

```diff
@@ -112,9 +107,6 @@ public void ModifyEnvironmentVariable(EnvironmentVariableTarget target)
         }
 
         [Theory]
```

Comment:
You removed all of the inputs from this theory.  The test will no longer run anything.